content/blog/getting-to-chatops-with-pulumi-webhooks/index.md

```diff
@@ -142,13 +142,12 @@ That's it! As stacks are created, updated, and so on, the webhook
 handler will be notified and you can start building any custom actions
 you can think of.
 
-For our initial release, there are four kinds of webhook events. But
+For our initial release, there are three kinds of webhook events. But
 we'll quickly be expanding coverage in the coming weeks.
```

Comment:
```suggestion
There are three kinds of webhook events.
```
Thoughts on removing the indication that future work is coming?

Reply:
Agreed.